.github/workflows/docs.yaml

```
@@ -0,0 +1,32 @@
+name: Documentation
+# This workflow builds the static documentation site, and publishes the results to GitHub Pages.
+# It runs on every push to the main branch, with changes in the docs and spec directories
+on:
+  workflow_dispatch:  # allow manual updates
+  push:
+    branches:
+      - main
```

Should this not be

```suggestion
      - master
```
?